DataFormats/SiStripCluster/interface/SiStripApproximateClusterv1.h

```diff
@@ -0,0 +1,31 @@
+#ifndef DATAFORMATS_SISTRIPAPPROXIMATECLUSTERv1_H
+#define DATAFORMATS_SISTRIPAPPROXIMATECLUSTERv1_H
+
+#include <vector>
+#include <numeric>
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
```

Comment:
I think they do no harm, but the vector and MessageLogger includes are not needed.